Updating server/database data

AnthonyDario opened the pull request:
Updated the database so that it now stores all of the data (resources, Elemari) the front-end wants. The server will now return this data upon login. I also updated the SaveState script so that it stores the new data when it pulls from the server.

AnthonyDario committed:
Updated the server and database data

Database now holds values for the four resources, the max
resources the player can hold, and the number of elemari the
player has.

AnthonyDario committed:
Cleaned up formatting of server code

AnthonyDario committed:
Server now returns resource on login

AnthonyDario committed:
New data is now saved into SaveState script

I also updated the server json to be more consistent.

CatalystOfNostalgia commented on server/models/user.py:
```diff
@@ -18,4 +18,15 @@ class User(Base):
+    maxAir = Column(Integer, default = 10) 
+    maxWater = Column(Integer, default = 10) 
+    maxEarth = Column(Integer, default = 10) 
+    fireEle = Column(Integer, default = 1)
```
Let's also add a poof count.

CatalystOfNostalgia commented on server/sql/create_tables.sql:
```diff
@@ -8,6 +8,18 @@ CREATE TABLE user(
 	email varchar(100) NOT NULL UNIQUE,
 	username varchar(100) NOT NULL UNIQUE,
 	password varchar(100),
+    fire int DEFAULT 0,
```
These tabs don't seem to be aligned properly.

AnthonyDario committed:
Added poof count to database

Poof count is now in database and is returned by the server and
stored by the game.

Also some naming was updated to be more consistent.

AnthonyDario committed:
fixed indentation

AnthonyDario commented:
Added the poof count and fixed style. Some people are using spaces and some people are using tabs for indentation...

CatalystOfNostalgia commented:
Merged @AnthonyDario
